css/style.css

```diff
@@ -170,4 +169,6 @@ body {
     background-color: rgba(255, 85, 0, 1);
 }
 
-   
\ No newline at end of file
+#b-margin{
```

Comment:
Everything seems fine, add an empty line at the end of the style.css for good practices.

Reply:
Please check made the changes needed.